packages/x-date-pickers/src/internals/demo/DemoContainer.tsx

```diff
@@ -76,7 +76,13 @@ export function DemoItem(props: DemoItemProps) {
   }
 
   return (
-    <Stack direction="column" alignItems="stretch" spacing={spacing} sx={sx}>
+    <Stack
+      direction="column"
+      alignItems="stretch"
+      spacing={spacing}
+      sx={sx}
+      className={textFieldClasses.root}
```

Comment:
Was there something that required adding this class here? 🤔 
The final result is a bit strange, where we end up with nested `MuiTextField-root` classes. 🙈 
![Screenshot 2023-02-16 at 11 16 39](https://user-images.githubusercontent.com/4941090/219324047-15a815b6-4b47-40fb-97eb-27face7f024c.png)

Reply:
And example of issue is https://next.mui.com/x/react-date-pickers/time-range-field/#uncontrolled-vs-controlled that doesn't receive the right min width because there is a wrapper, but I have reverted. We don't have to fix this in this PR.